dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/db/Sqlite.kt

```diff
@@ -0,0 +1,37 @@
+package org.jetbrains.kotlinx.dataframe.io.db
+
+import org.jetbrains.kotlinx.dataframe.io.TableColumnMetadata
+import org.jetbrains.kotlinx.dataframe.schema.ColumnSchema
+import java.sql.ResultSet
+import kotlin.reflect.typeOf
+
+/**
+ * Represents the Sqlite database type.
+ *
+ * This class provides methods to convert data from a ResultSet to the appropriate type for Sqlite,
+ * and to generate the corresponding column schema.
+ */
+public object Sqlite : DbType("sqlite") {
+    override fun convertDataFromResultSet(rs: ResultSet, tableColumn: TableColumnMetadata): Any? {
+        val name = tableColumn.name
+        return when (tableColumn.sqlType) {
+            "INTEGER", "INTEGER AUTO_INCREMENT" -> rs.getInt(name)
+            "TEXT" -> rs.getString(name)
+            "REAL" -> rs.getDouble(name)
+            "NUMERIC" -> rs.getDouble(name)
+            "BLOB" -> rs.getBytes(name)
+            else -> throw IllegalArgumentException("Unsupported SQLite type: ${tableColumn.sqlType}")
```

Comment:
I tried to read https://www.sqlitetutorial.net/sqlite-sample-database/
It failed with this error
`Unsupported SQLite type: NVARCHAR`.
There's also some unsupported type there that `ResultSet` sees as `DATE`
I think it's because SQLite doesn't truly has types and probably their names can vary greatly


I have several notes here:
1. I don't fully understand why interpret each column type from database metadata tables. I assume it has something to do with schema generation? JDBC interprets values and has information about how to read them: 
```
val type = rs.metaData.getColumnType(it)
when (type) {
java.sql.Types.FLOAT -> rs.getFloat()
...
}
```

2. Can we provide user with a way to support certain column types themselves? Like, if i tried to read that db without access to dataframe repository, i would have no way to fix the problem. Only to report to GitHub :( 

3. Probably worth to add column name to exception message

Reply:
1. The problem with java.sql.Types that many types in the SQL databases are mapped to Types.OTHER (Postgres, MariaDB, MSSQL) and better to control it manually, if you test it on your synthetic tests. If in the future the shorter schema will pass all my tests, I will replace on mapping with codes